XcodeServerSDK/API Routes/XcodeServer+Integration.swift

```diff
@@ -161,6 +161,38 @@ extension XcodeServer {
         }
     }
     
+    /**
+    XCS API call for fetching all commits for specific integration.
+    
+    - parameter integrationId: ID of integration.
+    - parameter success:       Optional Integration Commits object with result.
+    - parameter error:         Optional operation error.
+    */
+    public final func getIntegrationCommits(integrationId: String, completion: (integrationCommits: IntegrationCommits?, error: NSError?) ->()) {
+        
+        let params = [
+            "integration": integrationId
+        ]
+        
+        self.sendRequestWithMethod(.GET, endpoint: .Commits, params: params, query: nil, body: nil) { (response, body, error) -> () in
+            
+            guard error == nil else {
+                completion(integrationCommits: nil, error: error)
+                return
+            }
+            
+            guard let integrationCommitsBody = (body as? NSDictionary)?["results"] as? NSArray else {
+                completion(integrationCommits: nil, error: Error.withInfo("Wrong body \(body)"))
+                return
+            }
+            
+            let integrationCommits = IntegrationCommits(json: integrationCommitsBody[0] as! NSDictionary)
```

Comment:
Great question... I was asking myself the same one yesterday while writing this.

It's an `Array` but we don't know of this array will return more than one element - we know similiar things happened earlier.
If this can return more elements than we should change the name...

(I need a shrug emot shortcut...)